### PR TITLE
Update Amazon store dashboard

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates the Amazon store dashboard and related topics for the 4.0 release.

## Affected documentation pages

- Amazon onboarding home
- Amazon sales channel home
- Amazon store dashboard
- creating attributes
- learning and preparation
- store settings (review)

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

I did not do a staging build, but can start one if the reviewer wants.

### CODE REVIEW

File: src/sales-channels/asc/amazon-store-dashboard.md [hunk 1]
```diff
@@ -3,28 +3,48 @@ title: Amazon Store Dashboard
...
+The header of the store dashboard displays the same basic store information that is shown on the store card:
+
+- Store Name
+- Magento Website
+- Status
+- Created
+- Last Updated
+
+The store dashboard also includes store data and links to settings or more information:
+
```
Comment (leslietilling) [inline src/sales-channels/asc/amazon-store-dashboard.md]: I know it is hard to be consistent here, but it looks odd to have some of the named items linked and others to have links in the description. I would suggest having all links in the description.

File: src/sales-channels/asc/ob-store-review.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-## Advanced Store Settings
+## Store Settings
 
 1. On the _Admin_ sidebar, go to **Marketing** > _Channels_ > **Amazon**.
 
-1. Select the store you want to review and click <span class="btn">View Store</span>.
+1. Select the store you want to view and click <span class="btn">View Store</span>.
 
-1. On the store dashboard, click a store setting in the left-side menu.
+1. On the store's dashboard, click a store setting in the left-side menu.
```
Comment (leslietilling) [inline src/sales-channels/asc/ob-store-review.md]: I wouldn't use possessive, as it is bad form for non-person nouns. I've been trying to clean some of those up during editorial passes. The `store dashboard` works just fine.

File: src/sales-channels/asc/ob-store-review.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
```
Comment (leslietilling) [inline src/sales-channels/asc/ob-store-review.md]: Same issue with inconsistent linking for the named items here.

File: src/sales-channels/asc/ob-store-review.md (hunk 2 above)
Comment (hickskenh) [reply src/sales-channels/asc/ob-store-review.md]: will fix in the next PR.   well require a find and replace, this is used often.

File: src/sales-channels/asc/amazon-store-dashboard.md (hunk 1 above)
Comment (hickskenh) [reply src/sales-channels/asc/amazon-store-dashboard.md]: added link to store reports and logs, even though it requires additional text and clicks from the user.  the logs and reports have direct links in the bullet text

**Comment by hickskenh** [comment]
running tests

**Pull request merged**